addOns/pscan/CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
+### Added
+- Allow add-ons to obtain the auto tagging's tags.
```

Comment:
"...obtain the auto tagging's tags." should that be "tagger's" ?

or maybe just not posessive: "...the auto tagging tags" ?

Reply:
Tweaked.